scripts/build/index.js

```diff
@@ -0,0 +1,204 @@
+const fs = require('fs')
+const path = require('path')
+const chokidar = require('chokidar')
+const UglifyJS = require('uglify-js')
+const package = require('../../package.json')
+const options = {compress: {}}
+const strict = `'use strict'`
+
+const shared = [
```

Comment:
out of curiosity, why not prefer webpack?

Reply:
webpack does not produce flat-bundles.